dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -0,0 +1,165 @@
+probes can be found xref:../application_health.adoc#dev-guide-application-health[here].
+
+----
+$ oc set probe dc/nexus \
+	--liveness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	-- echo ok
+$ oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://nexus:8081/content/groups/public
+----
+
+[[nexus-adding-persistence-to-nexus]]
+=== Adding Persistence to Nexus
+
+[NOTE]
+====
+If you don't want persistent storage, continue to
+<<nexus-connecting-to-nexus,Connecting to Nexus>>. However, your cached
+dependencies will be lost if the pod is restarted for any reason.
+====
+
+Now we'll create a Persistent Volume Claim for Nexus, so that our dependencies
+will not be lost when the Pod running the server terminates. If you do not have
+admin access on your cluster, ask your system administrator to create a
+Read/Write Persistent Volume for you.  Otherwise, read
+xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in OpenShift] for instructions on
+creating a persistent volume.
+
+Add a Persistent Volume Claim to the Nexus Deployment Config.
+
+----
+$ oc volumes dc/nexus --remove --confirm
+$ oc volumes dc/nexus --add \
+	--name 'nexus-pvc' \
+	--type 'pvc' \
+	--mount-path '/sonatype-work/' \
+	--claim-name 'nexus-pv' \
+	--claim-size '1G'
+----
+
+This removes the previous ephemeral volume for the Deployment Config and adds a
+claim for 1 gigabyte of persistent storage mounted at `/sonatype-work`, which
+is where our dependencies will be stored. Due to the change in configuration,
+the Nexus pod will be redeployed automatically.
+
+Verify that Nexus is running by refreshing the Nexus page in your browser. You
+can monitor the deployment's progress using
+
+----
+$ oc get pods -w
+----
+
+[[nexus-connecting-to-nexus]]
+== Connecting to Nexus
+
+The next steps demonstrate defining a build that will use the new Nexus
+repository.  The rest of the tutorial will be using
+link:https://github.com/openshift/jee-ex.git[this example repository]
+with `wildfly-100-centos7` as a builder, but these changes should work for any
+project.
+
+The example builder image, wildfly-100-centos7, supports `MAVEN_MIRROR_URL` as
+part of its environment, so we can use this to point our builder image to our
+Nexus repository. If your image does not support consuming an environment
+variable to configure a Maven mirror, you may need to modify the builder image
+to provide the correct Maven settings to point to the Nexus mirror.
+
+----
+$ oc new-build openshift/wildfly-100-centos7:latest~https://github.com/openshift/jee-ex.git \
+	-e MAVEN_MIRROR_URL='http://nexus.<Nexus_Project>:8081/content/groups/public/'
+$ oc logs build/jee-ex-1 --follow
+----
+
+Replace `<Nexus_Project>` with the project name of the Nexus repository.  If it
+is in the same project as the application that is using it, you can remove the
+`<Nexus_Project>.`.  More information on DNS resolution in OpenShift can be
```

Comment:
Consider:

    xref:../../architecture/additional_concepts/networking.adoc#architecture-additional-concepts-networking[Learn more about DNS resolution in {product-title}].